CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.12)
```

Comment:
CMake 3.10 can't build this because of the use of `TRANSFORM` for lists [here](https://github.com/getsentry/crashpad/blob/getsentry/third_party/mini_chromium/CMakeLists.txt#L3), which is a CMake 3.12 addition: https://cmake.org/cmake/help/v3.12/release/3.12.html

Reply:
👍 We use `3.10` for the main project because of Android. Which we don’t support (yet) for crashpad so this is fine.